setup.py

```diff
@@ -226,7 +226,7 @@ def run(self):
         "cfgrib",
     ],
     tests_requires=["pytest", "pytest-qt", "pytest-mock"],
-    python_requires=">=3.8",
+    python_requires=">=3.8, <=3.11",  # limiting to 3.11 until ecmwflibs is not available for 3.12
```

Comment:
Is the extra space allowed? `>=3.8,<=3.11`?

Reply:
Should be yes, this https://packaging.python.org/en/latest/specifications/version-specifiers/#id5 says 

> Whitespace between a conditional operator and the following version identifier is optional, as is the whitespace around the commas.